packages/ui/src/components/app-sidebar/sidebar-search/sidebar-search.tsx

```diff
@@ -22,20 +16,17 @@ export function SidebarSearch(props: SidebarSearchProps) {
   const { setIsOpen } = searchContext
 
   return (
-    <Layout.Grid gapY="md">
-      {props.logo}
-      <button className="max-w-full overflow-hidden" onClick={() => setIsOpen(true)}>
-        <SearchInput
-          size="sm"
-          placeholder={t('component:navbar.search', 'Search')}
-          className="pointer-events-none"
-          inputContainerClassName="border-cn-borders-2 [&>.cn-input-prefix]:w-[34px] max-w-full overflow-hidden"
-          suffix={<Shortcut className="mr-1.5 transition-opacity group-data-[state=collapsed]:opacity-0">⌘K</Shortcut>}
-          readOnly
-          aria-hidden="true"
-          tabIndex={-1}
-        />
-      </button>
-    </Layout.Grid>
+    <button className="max-w-full overflow-hidden" onClick={() => setIsOpen(true)}>
```

Comment:
do we still need this button here? Can we use `SearchInput` alone and add `onClick`. Thoughts?